notification2-examples/shared-components/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>c8y.example.notification</groupId>
+        <artifactId>notification2-examples</artifactId>
+        <version>${revision}${changelist}</version>
+    </parent>
+
+    <artifactId>shared-components</artifactId>
```

Comment:
I don't like this naming, first of all it brings not good memories ;) and sounds that we can put there everything. What about naming it `notifications-client`? We can also rename `WebSocketClient` to `NotificationsClient` which will be consistent with module name containing client related code only.

Reply:
Maybe notifications-common is a good name (matching the package name)

Reply:
Will change it to notification2-example-client

Reply:
Done.